client/src/main/scala/drt/client/components/StaffAdjustmentDialogue.scala

```diff
@@ -98,7 +92,8 @@ object StaffAdjustmentDialogue {
           ^.defaultValue := applyRounding(defaultValue),
           ^.onChange ==> ((e: ReactEventFromInput) => {
             val newFromValue = e.target.value
-            scope.modState(callback(newFromValue))
+            val updatedState = callback(newFromValue)(state)
```

Comment:
does this need to be explicitly passed in?

Reply:
yes i believe so. the old line was a closure where scala does the passing of the argument to the result of callback(newFromValue) for you